### PR TITLE
Fixed issue #1689: Used correct target URL for command-click on brand…

### PR DESCRIPTION
**Pull request by DiegoAlbertoTorres**
… logo. Turned off 'autoindex' (directory listing) on nginx at (urlhash)s/beaker. Not sure why this was necessary, since 'autoindex' was already off at the root of the config file.

### CODE REVIEW

File: core/nginx/nginx.conf.template [hunk 1]
```diff
@@ -74,6 +74,7 @@ http {
       proxy_set_header Authorization "Basic %(auth)s";
     }
     location = /%(urlhash)s/beaker/ {
+      autoindex off;
```
Comment (samccone) [inline core/nginx/nginx.conf.template]: was this needed to fix the click issue or is this just a separate fix?

Comment (DiegoAlbertoTorres) [inline core/nginx/nginx.conf.template]: Separate fix I guess, but Scott noticed directory listings were still accessible thanks to the click issue.

Comment (samccone) [inline core/nginx/nginx.conf.template]: Word, sometimes it is nice to split these changes up into separate commits. Take for instance this case.

You open a PR with one commit that does 3 things. 

1 of the things causes a bug or regression.
Since it is all in one commit we can not git revert <SHA> without losing the other features.

If it is a single commit per "feature or change" it makes it easy to back out mistakes.

Comment (DiegoAlbertoTorres) [inline core/nginx/nginx.conf.template]: You're right. I'll keep it in mind!

File: core/src/main/web/app/controlpanel/controlpanel-directive.js [hunk 2]
```diff
@@ -36,7 +36,7 @@
 
         $scope.gotoControlPanel = function(event) {
           if (bkUtils.isMiddleClick(event)) {
-            window.open("./");
+            window.open("../../");
```
Comment (ildipo) [inline core/src/main/web/app/controlpanel/controlpanel-directive.js]: Can we use the bkHelper.getServerUrl() here?

Comment (DiegoAlbertoTorres) [inline core/src/main/web/app/controlpanel/controlpanel-directive.js]: bkHelper.serverUrl(x) gives a URL of the form 'ip:port/hash/x', but the click is supposed to redirect to the root ('ip:port/'). Is there another function that returns the base url without the hash?

**Comment by scottdraves** [comment]
:+1:
